src/ast/mod.rs

```diff
@@ -3000,12 +3001,10 @@ impl fmt::Display for Statement {
                     }
                 }
                 if *external {
-                    write!(
-                        f,
-                        " STORED AS {} LOCATION '{}'",
-                        file_format.as_ref().unwrap(),
-                        location.as_ref().unwrap()
-                    )?;
+                    if file_format.is_some() {
+                        write!(f, " STORED AS {}", file_format.as_ref().unwrap())?;
+                    }
```

Comment:
I think you can avoid this kind of unwrap by using this syntax:

```suggestion
                    if let Some(file_format) = &file_format {
                        write!(f, " STORED AS {file_format}")?;
                    }
```

As you have done below.

I'll make the change

Reply:
Apologies, and, yes, that is better.  Thank you for making that change @alamb !